ml-agents/mlagents/trainers/sac/models.py

```diff
@@ -44,6 +45,11 @@ def __init__(
         self.h_size = h_size
         self.activ_fn = self.swish
 
+        self.policy_memory_in: Optional[tf.Tensor] = None
```

Comment:
@ervteng These were generating warnings because they were being referenced without necessarily being created. Figured this was an easy fix.

There's another class of warnings about setting attributes outside of init, which I didn't try to tackle (yet)